desktop/core/src/desktop/js/parse/sql/generic/genericAutocompleteParser.Locations.test.js

```diff
@@ -46,6 +46,13 @@ describe('genericAutocompleteParser.js locations', () => {
     });
   };
 
+  it.skip('testing', () => {
+    assertLocations({
+      beforeCursor: 'select boo::INT, bar::int from customers; ',
+      expectedLocations: []
+    });
+  });
```

Comment:
Just a nice test to quickly see what the parser is returning, obviously can be deleted before merging